fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -1365,7 +1496,7 @@ void MultiVersionApi::loadEnvironmentVariableNetworkOptions() {
 	envOptionsLoaded = true;
 }
 
-MultiVersionApi::MultiVersionApi() : bypassMultiClientApi(false), networkStartSetup(false), networkSetup(false), callbackOnMainThread(true), externalClient(false), localClientDisabled(false), apiVersion(0), envOptionsLoaded(false) {}
+MultiVersionApi::MultiVersionApi() : bypassMultiClientApi(false), networkStartSetup(false), networkSetup(false), callbackOnMainThread(true), externalClient(false), localClientDisabled(false), apiVersion(0), envOptionsLoaded(false), threadCount(1) {}
```

Comment:
`threadCount` should be zero i.e. `(theadCount==0 && !localClientDisabled) || (theadCount > 0 && localClientDisabled)`. I think we should add check for this invariant, to make sure clients don't mess up with configuration.

Reply:
Agreed.  I haven't fixed this yet.  Also, the python client hangs if we set the thread count and don't provide a correct directory for the .so files.  Other than that, I think this is getting close to ready to merge.

Reply:
This is fixed now.  Also, support for the builtin client with single threaded external clients has been restored (but not carefully tested).